infrastructure/main.tf

```diff
@@ -24,23 +24,11 @@ module "servicebus-queue" {
   resource_group_name = "${azurerm_resource_group.rg.name}"
 }
 
-# save the queue's "listen" connection string to vault
-resource "vault_generic_secret" "servicebus-listen-conn-string" {
-  path = "secret/${var.vault_section}/cc/send-letter/servicebus-listen-conn-string"
-
-  data_json = <<EOT
-    {
-      "value": "${module.servicebus-queue.primary_listen_connection_string}"
-    }
-    EOT
-}
-
 module "db" {
-  source              = "git@github.com:contino/moj-module-postgres.git?ref=feature/specify-db-name"
-  product             = "${var.product}"
+  source              = "git@github.com:contino/moj-module-postgres.git?ref=master"
+  product             = "${var.product}-${var.microservice}-db"
```

Comment:
I didn't include microservice thinking the db belongs to send-letter rather send-letter-producer. Doesn't matter that much, though.

The change made to the DB might require a cleanup in aat.